src/api/.rubocop_todo.yml

```diff
@@ -818,6 +818,7 @@ Rails/ApplicationMailer:
 Rails/CreateTableWithTimestamps:
   Exclude:
     - 'db/migrate/20171030143054_create_kiwi_preference_types.rb'
+    - 'db/migrate/20240513154336_remove_watched_projects.rb'
```

Comment:
If we are sure we are not going to fix this, I would add this exclusion in the `.rubocop.yml` file, and not in the TODO file.

Reply:
Done!